src/shared/metadata/selectors.js

```diff
@@ -47,6 +47,15 @@ export const getSectionsByDataSetId = (metadata, dataSetId) => {
 export const getCategoryComboById = (metadata, id) =>
     getCategoryCombos(metadata)[id]
 
+/**
+ * Alphabetically sort the order of data elements on displayFormName
+ * @param {*} dataElements
+ */
+export const sortDataElementsByDisplayFormName = (dataElements) =>
```

Comment:
This doesn't really comply to the same API as the rest of the selectors, where we actually pass in the `metadata`.

I appreciate how easy this makes to test it though, but I don't think `selectors` is the right place for this. 

Could either move this and have this function in the `default-form.js`-file, or even make it inline (as a chained `.sort` on `selectors.getDataElementsByDataSetId(data, dataSet.id)`, even though that would make it more difficult to test. 

Another approach would be to make it an actual selector, something like this:

```javascript
export const getSortedDataElementsByDataSetId = createSelector(
    getDataElementsByDataSetId,
    (dataElements) =>
        dataElements.sort((deA, deB) =>
            deA.displayFormName?.localeCompare(deB.displayFormName)
        )
)
```

Reply:
Thats a good point thanks Birk. I wasn't sure if as a stand alone function this was a good place for it to live. I had pulled this out partly to make the test a bit more straight forward but looking at the usage of `getDataElementsByDataSetId` it does seem to be used only in `default-form.js` so I'll look at chaining the `.sort` in `getDataElementsByDataSetId`. I think I can get something place in the tests to support that easy enough.

Reply:
It's actually also used in `getDataElementsBySection`-selector, but that uses it only to find the elements and the order is determined by the `section.dataElements`. So it should be fine. At the same time you would do an unnecessary sort though. Not crucial, but I still think this should either be a different selector (like above), or just do it inline in `DefaultForm`.

Reply:
Ok I'll just go with doing it inline in the DefaultForm

Reply:
@Birkbjo Just pushed those changes. Gone with the name `getDataElementsByDataSetIdSortedOnDisplayFormName` its long but descriptive

Reply:
Chatting with @Birkbjo decide to shorten this to just `getDataElementsByDataSetIdSorted`